HARK/cstwMPC/cstwMPCold.py

```diff
@@ -637,7 +630,7 @@ def sensitivityAnalysis(parameter,values,is_time_vary):
         # If you want this to work, you must edit TractableBufferStockModel slightly.
         # See comments around line 34 in that module for instructions.
         if Params.do_tractable:
-            from TractableBufferStockModel import TractableConsumerType
+            from HARK.ConsumptionSaving.TractableBufferStockModel import TractableConsumerType
```

Comment:
I don't think there's anything wrong with this, but it isn't idiomatic Python. The comment doesn't really explain why it's a conditional import.